include/Game.h

```diff
@@ -9,8 +9,8 @@ extern "C" {
 #endif
 
 /// Pass pointer from gameInit to initialize BWAPI::BroodwarPtr
```

Comment:
Better append something like "`setGame()` must be called as part of the client init"